Makefile

```diff
@@ -133,7 +134,16 @@ install-go-test-tools:
 
 go-tests:
 	@mkdir -p tests/output
-	@gotestsum --junitfile tests/output/unit-tests.xml -- -race -coverprofile tests/output/cover.out $(GO_TEST_PACKAGES)
+	@for package in $(GO_TEST_PACKAGES); do \
+		if [ "$$package" = "./components/producers/golang-nancy/examples" ]; then \
```

Comment:
We skip this as in this path there's an example vulnerable `go.mod`

Reply:
can't we use a `-not` flag in the find command above to exclude this?

Reply:
Fair, I amended the command to ignore the path :)